modules/platforms/dotnet/Apache.Ignite.Tests/FakeServer.cs

```diff
@@ -309,6 +309,7 @@ protected override void Handle(Socket handler, CancellationToken cancellationTok
 
                     case ClientOp.TxBegin:
                         reader.Skip(); // Read only.
+                        reader.Skip(8); // TimeoutMillis.
```

Comment:
```suggestion
                        reader.Skip(); // TimeoutMillis.
```

`8` means skip 8 values, not 8 bytes.